src/components/SQLEditor/SQLEditor.tsx

```diff
@@ -331,12 +340,16 @@ function SQLEditor({
         />
       </div>
 
-      <div className="w-full flex items-center justify-between mt-2">
+      {error && name && (
```

Comment:
What is the name for?

Reply:
Wouldn't it be better to pass an error param that takes JSX? Something like:
```jsx
<SQLEditor error={<span>My error</span>} /> 
```

I imagine that would be more future proof...